Shared/mods/deathmatch/logic/CScriptDebugging.cpp

```diff
@@ -156,12 +157,12 @@ void CScriptDebugging::LogCustom(lua_State* luaVM, const char* szMessage)
 }
 
 void CScriptDebugging::LogString(const char* szPrePend, const SLuaDebugInfo& luaDebugInfo, const char* szMessage, unsigned int uiMinimumDebugLevel,
-                                 unsigned char ucRed, unsigned char ucGreen, unsigned char ucBlue)
+                                 unsigned char ucRed, unsigned char ucGreen, unsigned char ucBlue, bool bOmitDebugInfo)
 {
     SString strText = ComposeErrorMessage(szPrePend, luaDebugInfo, szMessage);
 
-    // Create a different message if type is "INFO"
-    if (uiMinimumDebugLevel > 2)
+    // Create a different message if type is "INFO" or it's a custom message with debug lines output omitted
+    if (uiMinimumDebugLevel > 2 || bOmitDebugInfo)
         strText = SString("%s%s", szPrePend, szMessage);
```

Comment:
Should be if-else here so we don't call `ComposeErrorMessage` if the if statement below (line 165) is true.